activesupport/lib/active_support/cache.rb

```diff
@@ -352,9 +352,14 @@ def read_multi(*names)
       #
       # Returns a hash with the data for each of the names. For example:
       #
-      #   cache.write("bim", "bam")
-      #   cache.fetch_multi("bim", "boom") { |key| key * 2 }
-      #   # => { "bam" => "bam", "boom" => "boomboom" }
+      #   cache.write("new_key", "value")
+      #   cache.fetch_multi("new_key", "unknown_key") do |key|
```

Comment:
Calling it "new_key" makes it more confusing here. As the key is known it's not new.